docs/asciidoc/modules/ROOT/pages/installation/index.adoc

```diff
@@ -22,7 +22,7 @@ The version compatibility matrix explains the mapping between Neo4j and APOC ver
 [opts=header]
 |===
 |apoc version | neo4j version
-| http://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/5.0.0.0[5.0.0.0^] | 5.0.0 (5.0.x)
+| http://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/5.1.0[5.1.0^]| 5.1.0 (5.1.x)
```

Comment:
5.2 has also been released, maybe that can be added? :)

Reply:
hmm, all of these links should be https as well, not sure why they're not

Reply:
And just to be annoying, can you remove all versions below 4.2, we don't support them anymore 🙈